sdk/core/azure-core/README.md

```diff
@@ -18,6 +18,37 @@ APIs in one client library, you will know how to use them in other client librar
 - A [Java Development Kit (JDK)][jdk_link], version 8 or later.
 
 ### Include the package
+#### Include the BOM file
```

Comment:
We should probably move this paragraph here.
```suggestion

Typically, you won't need to install or depend on Azure Core, instead it will be transitively downloaded by your build tool when you depend on client libraries using it.

#### Include the BOM file
```

Reply:
I wrote a tool that added these lines :). Based on the context to this file, I completely agree :). Thanks!